src/components/dashboard/Dashboard.js

```diff
@@ -14,9 +15,11 @@ const Dashboard = () => (
                 <span className="writing">Writing.</span>
                 <span className="intro-text">You are home</span>
               </p>
-              <button className="btn btn-success dash-btn" type="button">
-                { "Let's Write" }
-              </button>
+              <NavLink to="/create-article/">
```

Comment:
This is not a NavLink. Use Link instead

Reply:
@dondrzzy NavLink has been modified to Link